packages/iTwinUI-react/src/core/Tabs/Tabs.test.tsx

```diff
@@ -113,7 +114,7 @@ it('should not fail with invalid active tab and set the closest one', () => {
   expect(tabs[2].className).toContain('iui-tab iui-active'); // 2 is closest to 100
 });
 
-it('should render strings in HorizontalTab child component', () => {
+it('should render strings in child component', () => {
```

Comment:
Removing "HorizontalTab" from test name because I think that was left in when `VerticalTabs` was added in PR [#278](https://github.com/iTwin/iTwinUI-react/pull/278). Same on [L131](https://github.com/iTwin/iTwinUI-react/pull/908/files#diff-1d4cb35c3084fbd5724c07e7c967c0973a84088435307146648e9380330f5d17R131)